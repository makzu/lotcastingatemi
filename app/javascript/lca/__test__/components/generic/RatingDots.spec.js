// @flow
import renderer from 'react-test-renderer'

import RatingDots from '../../../components/generic/ratingDots.jsx'

test('RatingDots renders at all', () => {
  const component = renderer.create(<RatingDots rating={3} />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
