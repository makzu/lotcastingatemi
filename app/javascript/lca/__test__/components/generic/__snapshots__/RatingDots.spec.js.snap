// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RatingDots renders at all 1`] = `
<div
  aria-label={3}
>
  <div
    className="RatingDots-fullDot-2"
  >
    ●
  </div>
  <div
    className="RatingDots-fullDot-2"
  >
    ●
  </div>
  <div
    className="RatingDots-fullDot-2"
  >
    ●
  </div>
  <div
    className="RatingDots-emptyDot-1"
  >
    ○
  </div>
  <div
    className="RatingDots-emptyDot-1"
  >
    ○
  </div>
</div>
`;
